packages/@aws-cdk/aws-ecs/README.md

```diff
@@ -123,6 +123,31 @@ cluster.addAutoScalingGroup(autoScalingGroup);
 
 If you omit the property `vpc`, the construct will create a new VPC with two AZs.
 
+## Spot Instances
+
+To add spot instances into the cluster, you must specify the `spotPrice` in the `ecs.AddCapacityOptions` and optionally enable the `spotInstanceDraining` property.
+
+```ts
+// Add an AutoScalingGroup with spot instances to the existing cluster
+cluster.addCapacity('AsgSpot', {
+  maxCapacity: 2,
+  minCapacity: 2,
+  desiredCapacity: 2,
+  instanceType: new ec2.InstanceType('c5.xlarge'),
+  spotPrice: '0.0735',
+  // Enable the Automated Spot Draining support for Amazon ECS
+  spotInstanceDraining: true,
+});
+
+// And optionally add another AutoScalingGroup with on-demand instances to the same cluster
```

Comment:
Also, since setting up the autoscaling group has been introduced in the example above (in Clusters section), this example's value is limited.

Reply:
fixed https://github.com/aws/aws-cdk/pull/4360/commits/45bef1da7735bc55ff50668e70c7c3c9349b6944